sdk/keyvault/keyvault-keys/README.md

```diff
@@ -40,6 +40,18 @@ To quickly create the needed Key Vault resources in Azure and to receive a conne
 
 [![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
 
+To fill this form:
+
+1. If you know about ARM templates, or you're interested to learn about them, we recommend you to take some time to read through the [ARM template used by the deploy button in this readme](https://github.com/Azure/azure-sdk-for-js/blob/keyvault/deploy-button-guide/sdk/keyvault/test-resources.json). You can read more about ARM templates in the [Azure Resource Manager templates overview](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview).
```

Comment:
I'm not sure about this bullet point - users shouldn't feel like they have to learn a super complex topic just to use the simple deployment form. Also this is the readme so we should prefer short and sweet!

Reply:
Ok! What do you think if we move similar to this to the end of the bullet points? Like "If you want to learn more about.... please read...."

Reply:
A link anywhere is fine by me if it's short. Like "Learn more about [arm templates]()."

Reply:
I moved this to another file, just in this same branch, but this PR seems stale. Are you able to see this commit as part of this PR? https://github.com/sadasant/azure-sdk-for-js/commit/48aa4b909c27692322c1ff00a1d9eb1f2f88a436